hititer.go

```diff
@@ -122,7 +122,10 @@ func (d *indexData) trigramHitIterator(ng ngram, caseSensitive, fileName bool) (
 	iters := make([]hitIterator, 0, len(variants))
 	for _, v := range variants {
 		if fileName {
-			blob := d.fileNameNgrams[v]
+			blob, err := d.fileNameNgrams.GetBlob(v)
```

Comment:
This should be a follow up, but could we add a GetBlob to d.ngrams to simplify the code below / combine the two code paths?